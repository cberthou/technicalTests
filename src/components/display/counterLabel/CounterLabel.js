import React from 'react';

const CounterLabel = ({ children }) => (
    <span>{children}</span>
);

export default CounterLabel;
