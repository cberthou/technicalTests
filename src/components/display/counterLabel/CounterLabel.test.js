import React from 'react';
import { shallow } from 'enzyme';
import CounterLabel from './CounterLabel';

describe('src/components/display/counterLabel', () => {
    it('should display the label', () => {
        const wrapper = shallow(<CounterLabel count="1234" />);

        expect(wrapper).toMatchSnapshot();
    });
});
