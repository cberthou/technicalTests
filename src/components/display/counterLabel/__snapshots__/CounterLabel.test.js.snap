// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/display/counterLabel should display the label 1`] = `<span />`;
